citesphere/pom.xml

```diff
@@ -25,7 +25,7 @@
         <spring.kafka.version>2.2.6.RELEASE</spring.kafka.version>
         <spring-social-zotero.version>0.12</spring-social-zotero.version>
         <citesphere.messages.version>0.4</citesphere.messages.version>
-        <citesphere.model.version>1.19</citesphere.model.version>
+        <citesphere.model.version>1.22-SNAPSHOT</citesphere.model.version>
```

Comment:
is there another pull request this depends on? if so, please link? if not, this is not a valid version.

Reply:
No, there's no other PR on which this depends on. This is the latest version of citesphere-model repository's develop branch. I have reverted this back, as it's not needed here.